doc/users/project/mission.rst

```diff
@@ -0,0 +1,27 @@
+Matplotlib Mission Statement
+============================
+
+The mission of the Matplotlib developer community is to develop, maintain, and
+support Matplotlib and its extensions which provide plotting tools for users of
+the Scientific Python Ecosystem.  We success is based on how effectively we
+enable our users, across scientific, industrial, commercial, and educational
+applications, to achieve their goals.  A diverse developer community creates
+the best software and we welcome anyone, regardless of experience, education,
+gender, or nationality, who shares our mission to the community.
+
+
+Adapting the requirements :ref:`laid out by John Hunter
+<History>` Matplotlib should:
+
+* Support all users of the Scientific Python Ecosystem;
+* Produce high-quality raster and vector formats output;
+* Be embeddedable in graphical user interfaces for application development;
+* Be understandable and extendable by scientists, data scientists, and
+  engineers;
+* Make standard plots easy;
```

Comment:
```suggestion
* Make common cross domain plots easy;
```
 standard is domain specific

Reply:
fair, but I think that "standard" is still clear and "common cross domain" is too jargonny.

Reply:
I think "cross domain" is pretty jargony, and I'm not even sure we didn't coin it ourselves, as Matplotlib is the only context I have heard it used for.

Reply:
Why not just "common plots"?

Reply:
Yes, "common plots".